theme/base.scss

```diff
@@ -0,0 +1,19 @@
+
+// Guss-rem calculations assume a 10px baseline on the `:root`.
+// Rather than enforcing this by setting `$guss-rem-baseline` in pasteup-forms' source,
+// which could potentially be in duplicate for real implementations,
+// these styles set the baseline for the Cortana visual styleguide.
+
+html {
+    // Set baseline font size to 10px
+    // This is used as a baseline for rem (root ems) values
+    // Calc needed for IE11 to do the math properly
+    // See http://bit.ly/1g4X0bX — thanks @7studio and @dawitti
+    font-size: -webkit-calc(1em * .625);
```

Comment:
If we can remove the (few) instances of font-sizes in forms we can probably remove the need for this.